docs/conf/portland/2021/news/announcing-speakers.rst

```diff
@@ -0,0 +1,49 @@
+:template: {{year}}/generic.html
+
+.. post:: February 14, 2021
+   :tags: portland-2021, speakers, tickets, visiting, sponsors
+
+Announcing speakers and sessions
+================================
+
+We're writing today with an update on the fast-approaching Portland conference!
```

Comment:
```suggestion
We're writing today with an update on the fast-approaching Portland conference.
```
Maybe not have every sentence end with an exclamation mark here.

Reply:
Sounds good!